### PR TITLE
[ticket/14695] Add posting_editor_subject_prepend/append template events

### PR DESCRIPTION
**Pull request by imkingdavid**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-14695

PHPBB3-14695

### CODE REVIEW

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -1487,6 +1487,13 @@ posting_editor_subject_after
 * Since: 3.1.0-a2
 * Purpose: Add field (e.g. textbox) to the posting screen after the subject
 
+posting_editor_subject_append
+===
+* Locations:
+    + styles/prosilver/template/posting_editor.html
+* Since: 3.1.10
```
Comment (Crizz0) [inline phpBB/docs/events.md]: 3.1.10-RC1 (afaik)

**Comment by marc1706** [comment]
Is there a reason you didn't add this to subsilver2 as well?

**Comment by imkingdavid** [comment]
Didn't know we still had to. I'll add it when I have a chance.

**Comment by VSEphpbb** [comment]
I'd suggest putting these on to new lines:
```html
<dd>
    <!-- EVENT posting_editor_subject_prepend -->
    <input type="text" name="subject" id="subject" size="45" maxlength="<!-- IF S_NEW_MESSAGE -->120<!-- ELSE -->124<!-- ENDIF -->" tabindex="2" value="{SUBJECT}{DRAFT_SUBJECT}" class="inputbox autowidth" />
    <!-- EVENT posting_editor_subject_append -->
</dd>
```
The line is already ridiculously long as it is

**Comment by imkingdavid** [comment]
@marc1706 added subsilver2 changes

@VSEphpbb added newlines for each event.

**Pull request merged**